service-proxy.js

```diff
@@ -376,13 +385,15 @@ function handleRequest(req, buildRes) {
         return;
     }
 
-    if (self.isBlocked(req.headers && req.headers.cn, req.serviceName)) {
+    var rd = req.headers && req.headers.rd;
```

Comment:
can we call this actually `routingDelegate` and then add a `var nextService = routingDelegate || req.serviceName;` and use it throughout?